src/irsForms/F1040.ts

```diff
@@ -169,6 +169,11 @@ export default class F1040 implements Form {
   standardDeduction = (): number => {
     if (this.filingStatus === undefined) {
       return 12400
+    } else if (this.isTaxpayerDependent || this.isSpouseDependent) {
+      return Math.min(
+        (F1040.standardDeductions[this.filingStatus] ?? 12400),
```

Comment:
I think all these ###s should be parameters in the federal.ts file. It's painful to imagine them changing year to year, but in general looking them up somewhere feels better.

Reply:
Good idea, that will help keep the code clean and easy to debug